src/main/api/2/studio-api-2.yaml

```diff
@@ -2988,6 +2994,244 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /content/children_by_id:
+    get:
+      tags:
+        - content
+      summary: Get list of children for given item id
+      description: 'Required permission "get_children" ("site member")'
+      operationId: getChildrenById
+      parameters:
+        - name: siteId
+          in: query
+          description: Site ID
+          required: true
+          schema:
+            type: string
+        - name: id
+          in: query
+          description: item id to get children for
+          required: true
+          schema:
+            type: string
+        - name: locale
+          in: query
+          description: filter children by locale, or show source items (not translation) if translatation to specified locale does not exist
+          schema:
+            type: string
+        - name: sortStrategy
+          in: query
+          description: >-
+            sort strategy (sort order)
+            Possible values:
+              - default (order by: navigation pages, floating pages, folders, level descriptor)
+              - alphabetical
+              - folders first
```

Comment:
What's the parameter here? foldersFirst?

Explain that this is alphabetical folders first, then alphabetical files after.